### PR TITLE
Give a Talk Page

### PR DESCRIPTION
**Pull request by shrunyan**
Refreshing the give a talk content. Adding prep recommendations.

### CODE REVIEW

**Comment by hbrysiewicz** [comment]
+1

File: give-a-talk.md [hunk 1]
```diff
@@ -1,40 +1,30 @@
...
 
-Are you thinking of giving a short lightning talk?
+Once you are ready to submit a talk please email your talk subject to the [SDJS organizers](mailto:sandiegojs-organizers@googlegroups.com).
 
-Comment on an event page or, better yet, contact
-[the organizers](mailto:sandiegojs-organizers@googlegroups.com) directly.
+#### Preparing for Your Talk
 
-Don't know what you want to talk about yet?
+You will be presenting on a `600px` width projector so make sure to tailor your slides to that dimension. Using a large font with white text on a black background will make your content more legible.
```
Comment (treyhunner) [inline give-a-talk.md]: 800px width (not 600px width).  "800x600" works too.

Comment (treyhunner) [inline give-a-talk.md]: Black on white text, not white on black.  I got this suggestion from a Rebecca Murphey tweet sometime in the past but I can't find the tweet or blog post explaining it.

File: give-a-talk.md [hunk 2]
```diff
@@ -1,40 +1,30 @@
...
-Want to show off a niche feature of one of your favorite libraries?
+We also have a wiki page with [talk ideas][1] that you can use for inspiration.
 
-If you have something JavaScript-related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-### How to give a talk
+#### Submitting your talk
 
-Are you thinking of giving a short lightning talk?
+Once you are ready to submit a talk please email your talk subject to the [SDJS organizers](mailto:sandiegojs-organizers@googlegroups.com).
```
Comment (treyhunner) [inline give-a-talk.md]: Maybe we should offer to have them reach out to even just chat about what they *could* give a talk on? I chatted with a couple folks last night who didn't think they knew what to talk about but after talking with them for a minute we realized they had lots of topics they could talk about.

File: give-a-talk.md [hunk 3]
```diff
@@ -1,40 +1,30 @@
...
-[the organizers](mailto:sandiegojs-organizers@googlegroups.com) directly.
+#### Preparing for Your Talk
 
-Don't know what you want to talk about yet?
+You will be presenting on a `600px` width projector so make sure to tailor your slides to that dimension. Using a large font with white text on a black background will make your content more legible.
 
-We have a wiki page with [talk ideas][] that you can use for inspiration.
+Want to chat about your talk while you prepare it?  Join the [chat room][2]!
 
-Want to chat about your talk while you prepare it?  Join [the chat room](/chat-room.html)!
```
Comment (treyhunner) [inline give-a-talk.md]: Maybe we should have a do's and don'ts section.

- Do use black on white slides
- Don't use more than a small number of slides (maybe 6 preferably?)
- Don't try to do a live demo or write code or do anything but slides for your first talk... it's really hard for a lightning talk

We could also link to lightning talks we think are good examples.

**Comment by treyhunner** [comment]
@shrunyan :sparkles: Thanks for doing this rewrite.  You're awesome!

File: give-a-talk.md (hunk 1 above)
Comment (shrunyan) [inline give-a-talk.md]: @treyhunner to clarify, your saying black text on a white slide is the desired colors?

File: give-a-talk.md (hunk 3 above)
Comment (shrunyan) [inline give-a-talk.md]: I really like this suggestion! Going to reformat it to this style.

File: give-a-talk.md (hunk 2 above)
Comment (shrunyan) [inline give-a-talk.md]: I had originally put more avenues of contact but thought might be a little choice paralysis versus a singular option.

File: give-a-talk.md [hunk 4]
```diff
@@ -1,40 +1,46 @@
...
+Have you been learning JavaScript and want to spend a couple minutes sharing something you learned? Did you find a library or discover a JS feature you'd like to share? Have an interesting method for learning or teaching JavaScript? Want to share something you made? Want to show off a niche feature of one of your favorite libraries?
 
-Did you find a library or discover a JS feature you'd like to share?
+We also have a wiki page with [talk ideas][1] that you can use for inspiration.
 
-Have an interesting method for learning or teaching JavaScript?
+---
 
-Want to share something you made?
+__*If you are unsure what to talk about do not hesitate to jump onto one of the [chat channels][2] and ask about presenting.*__
```
Comment (shrunyan) [inline give-a-talk.md]: @treyhunner adding a chat callout for people who are not sure what to talk about.

File: give-a-talk.md [hunk 5]
```diff
@@ -1,40 +1,46 @@
 ---
 layout: page
-title: Give a Lightning Talk
+title: We want you to give a talk!
 ---
 {% include JB/setup %}
 
-### We want you to give a talk!
+If you have something JavaScript related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
```
Comment (kball) [inline give-a-talk.md]: Do we want to call out that if you have a longer talk as well we might still have an opening?  Or does that complicate things needlessly?

**Comment by kball** [comment]
This looks good, thanks for tackling this @shrunyan

File: give-a-talk.md [hunk 6]
```diff
@@ -1,40 +1,50 @@
...
+_Want to chat about your talk while you prepare it?  Join the [chat room][2]._
 
 ### Suggest a presentation topic
 
-Have an idea you'd like to hear someone give a talk about?  We have a wiki page with [talk ideas][].  Go add your ideas to it!
+Have an idea you'd like to hear someone give a talk about?  We have a wiki page with [talk ideas][1] you can clone and add your ideas to it! Alternatively you can post suggestions to any of the [chat rooms][2].
+
+### Extended Talks
+
+If you have a longer talk we will happily accept those as well.
```
Comment (shrunyan) [inline give-a-talk.md]: @kball added section for extended talk notes. We can expand upon this as needed.

Comment (treyhunner) [inline give-a-talk.md]: We don't have a policy around this yet.  I suggest preferring extended talks from speakers who have given good lightning talks before.

**Comment by treyhunner** [comment]
:+1: I made a comment about encouraging lightning talks before extended talks. Looks good besides that!

**Comment by treyhunner** [comment]
:boom: :fireworks: :ship:

**Pull request merged**